File: CS/DialogsDemo.Wpf/Helpers/DialogsDemoHelper.cs
using DevExpress.Internal;
using System.IO;

namespace DialogsDemo.Helpers {
    public static class DialogsDemoHelper {
        public static string GetDialogsDataPath(string relativePath) {
            return Path.GetFullPath(DataDirectoryHelper.GetFile("Dialogs\\" + relativePath, DataDirectoryHelper.DataFolderName));
        }
        public static string GetPhotosPath(string relativePath) {
            return Path.GetFullPath(DataDirectoryHelper.GetFile("Photos\\" + relativePath, DataDirectoryHelper.DataFolderName));
        }
        public static Stream GetDataStream(string dataFileName) {
            string path = GetDialogsDataPath(dataFileName);
            return File.OpenRead(path);
        }
    }
}

File: VB/DialogsDemo.Wpf/Helpers/DialogsDemoHelper.vb
Imports DevExpress.Internal
Imports System.IO

Namespace DialogsDemo.Helpers
    Public NotInheritable Class DialogsDemoHelper

        Private Sub New()
        End Sub

        Public Shared Function GetDialogsDataPath(ByVal relativePath As String) As String
            Return Path.GetFullPath(DataDirectoryHelper.GetFile("Dialogs\" & relativePath, DataDirectoryHelper.DataFolderName))
        End Function
        Public Shared Function GetPhotosPath(ByVal relativePath As String) As String
            Return Path.GetFullPath(DataDirectoryHelper.GetFile("Photos\" & relativePath, DataDirectoryHelper.DataFolderName))
        End Function
        Public Shared Function GetDataStream(ByVal dataFileName As String) As Stream
            Dim path As String = GetDialogsDataPath(dataFileName)
            Return File.OpenRead(path)
        End Function
    End Class
End Namespace
